BranchMetrics/ios-branch-deep-linking-attribution
# [SDK-1944][patch] Add Developer ID to v1/install

## Reference
SDK-1944 -- [iOS] Add Developer ID to v1/install

## Summary
This PR adds the identity field to the `v1/install` endpoint by utilizing a temporary variable to store the identity set by the `setIdentity()` method. This is necessary because the `v1/profile` endpoint is called after the `v1/install` endpoint.

## Motivation
The addition of the identity field to the `v1/install` endpoint aims to achieve parity between the iOS and Android SDKs. This improvement has also been requested by clients.

## Type Of Change


## Testing Instructions

1. Call the `setIdentity()` method.
2. Run the app as a fresh install.
3. Observe the identity field in the `v1/install` request, as well as in subsequent `v1/open` requests as expected.
4. Ensure that the `v1/profile` endpoint is still called as usual after the `v1/install` endpoint.










cc @BranchMetrics/saas-sdk-devs for visibility.
